rfid2/rfid2.ino

```diff
@@ -0,0 +1,85 @@
+#include <Keyboard.h>
+#include <SPI.h>
+#include <MFRC522.h>
+
+#define SS_PIN 10
+#define RST_PIN 5
+#define KEY_RETURN 0xB0  // The hex value for the return key is 0xB0.
+
+MFRC522 mfrc522(SS_PIN, RST_PIN);
+char Enter = KEY_RETURN;  // Return key is declared as Enter.
+String readid;
```

Comment:
It is very confusing if you use the name of an array and call it String.

Reply:
Please Elaborate.

Reply:
oh sorry, I read this bit wrong. My fault. it's fine